### DISCUSSION TITLE
Breaking Change: Default Debian version set to Debian 11

### DISCUSSION THREAD

**Discussion by mthalman** [dotnet/dotnet-docker#3696] [Announcements]
# Breaking Change: Default Debian version set to Debian 11

The default Debian version has been upgraded to Debian 11 (Bullseye) as part of the .NET 6 release. This means that any usage of the `6.0`, `6.0.x`, or `latest` tags when targeting Linux containers will give you an image based on Debian 11.

## Details

This upgrade conforms to our [policy of releasing new major versions of .NET on the latest stable version of Debian](https://github.com/dotnet/dotnet-docker/blob/main/documentation/supported-platforms.md).

Previous .NET versions (.NET Core 3.1, .NET 5) will continue to use Debian 10 (Buster) as the default but [image tags](https://hub.docker.com/_/microsoft-dotnet) are available for Debian 11 for these versions. Container images based on Debian 10 will not be made available for .NET 6.

If you run into unexpected issues as part of migrating from .NET 5 to .NET 6 with the use of Debian, it is recommended to read the [Debian 11 release notes](https://www.debian.org/releases/bullseye/releasenotes) to understand the impact of this upgrade.

## Related Issue

https://github.com/dotnet/dotnet-docker/issues/2576

**Comment by danigian** [comment]
@mthalman can you please confirm that, even though no docker images are available based on Debian 10 (buster) for .NET 6, Debian 10 is anyways a supported OS for .NET 6 (as stated [here](https://github.com/dotnet/core/blob/main/release-notes/6.0/supported-os.md))?

**Comment by mthalman** [comment]
@danigian - That's correct. The OS versions that .NET supports is a superset of the official container images that we provide. If it's listed as an OS version that .NET supports, that includes support within containers.
